includes/definitions/discovery/timos.yaml

```diff
@@ -82,7 +82,7 @@ modules:
                         - { value: 8, generic: 0, graph: 0, descr: GreenBlink }
                         - { value: 9, generic: 2, graph: 0, descr: RedBlink }
                 -
-                    oid: tmnxChassisTable
+                    oid: tmnxChassisOverTempState
```

Comment:
Hi
What was wrong with the initial oid ? Looks good according to the MIB file at least.

Reply:
See if I can explain it...

The tmnxChassisTable has 10 different states for alarms and in the same table, the tmnxChassisOverTempState has only 2 states.  When discovery happens, it overwrites the values 1 and 2 in the state translation table.

Before:  The Critical LED would say "Overtemp"
![image](https://user-images.githubusercontent.com/59381293/115221874-773d6c80-a14d-11eb-9ef5-6363e5cd31aa.png)
With the patch, the correct alarm text is displayed "Red :
![image](https://user-images.githubusercontent.com/59381293/115222090-b1a70980-a14d-11eb-9a1f-25abdadca70f.png)

Reply:
Got it.
The issue should be solved in a different way. 
Just keep current OID and VALUE, and add this statement to ensure the state is correct for this entry. : 
` state_name: tmnxChassisOverTempState `
Basically, any state from this table without a `state_name` defined should be corrected.